GUIDE-zh.md

```diff
@@ -274,20 +260,70 @@ val paymentSession = AirwallexPaymentSession.Builder(
 ## 低层API集成
 你可以基于我们的低层API来构建完全由你自定义的UI。
 
-### 步骤一：创建AirwallexSession和Airwallex对象
+### 添加低层API集成依赖
+Airwallex Android SDK 支持Android API 21及以上版本。
+
+- 安装SDK
+  已经上传到[Maven Central](https://repo1.maven.org/maven2/io/github/airwallex/), 你只需要添加Gradle依赖项。
+
+在项目的根目录下，打开`build.gradle`，并添加以下内容：
+
+```groovy
+    dependencies {
+        // It's required
+        implementation 'io.github.airwallex:payment-components-core:4.5.0'
+        
+        // Select the payment method you want to support.
+        implementation 'io.github.airwallex:payment-card:4.5.0'
+        implementation 'io.github.airwallex:payment-redirect:4.5.0'
+        implementation 'io.github.airwallex:payment-wechat:4.5.0'
+        implementation 'io.github.airwallex:payment-googlepay:4.5.0'
+    }
+```
+
+### 配置低层API SDK
+设置SDK后，需要使用一些参数来配置SDK。 在使用Airwallex SDK confirm PaymentIntent并完成付款之前，您应在自己的服务器中创建PaymentIntent，以确保在自己的系统中维护信息
+#### 初始化SDK
+
+我们提供了一些可用于调试SDK的参数，你可以在Application中调用
 ```kotlin
-val session = buildSession(paymentIntent, customerId)
-val airwallex = Airwallex(this@PaymentCartFragment)
+     AirwallexStarter.initialize(
+        application,
+        AirwallexConfiguration.Builder()
+            .enableLogging(true)                // Enable log in sdk, and don’t forogt to set to false when it is ready to release
+            .setEnvironment(Environment.DEMO)   // You can change the environment to STAGING, DEMO or PRODUCTION. It must be set to PRODUCTION when it is ready to release.
+            .setSupportComponentProviders(
+                listOf(
+                    CardComponent.PROVIDER,
+                    WeChatComponent.PROVIDER,
+                    RedirectComponent.PROVIDER,
+                    GooglePayComponent.PROVIDER
+                )
+            )
+            .build(),
+        ExampleClientSecretProvider()           // If you need to support recurring, you must to support your custom ClientSecretProvider
```

Comment:
`you must to support` -> `you must provide`, could you please also correct everywhere else this sentence is used?

Reply:
ok